roles/openshift_master/defaults/main.yml

```diff
@@ -6,7 +6,9 @@ os_firewall_allow:
 - service: etcd embedded
   port: 4001/tcp
 - service: api server https
-  port: 8443/tcp
+  port: "{{ openshift.master.api_port }}/tcp"
```

Comment:
I'm actually surprised this works, but the last time I tested I don't think we were setting openshift_facts for masters in the playbooks for the certificate generation.